discretisedfield/mesh.py

```diff
@@ -40,12 +34,6 @@ class Mesh:
     discretisation cells ``n`` in all three dimensions. Either ``cell`` or
     ``n`` can be passed, not both.
 
```

Comment:
Should we briefly mention the attribute `bc` here? ("can be used to define different types of boundary conditions; string")